### ISSUE TITLE
get_discussion_id() errors due to row combine procedure

### ISSUE THREAD

**Issue by coatless** [daranzolin/rcanvas#44]
While attempting to retrieve discussion components with `get_discussion_id()`, I'm receiving an error of:

```r
get_discussion_id(discussion_id, object_id)
Error: Argument 5 is a list, must contain atomic vectors
```

The API call issues is found under [get a single topic](https://canvas.instructure.com/doc/api/discussion_topics.html#method.discussion_topics_api.show) with structure:

```
/api/v1/groups/:group_id/discussion_topics/:topic_id
```

The error is being triggered inside `rcanvas:::process_response()`

Specifically, when it attempts to bind multiple data frames together, e.g.

https://github.com/daranzolin/rcanvas/blob/7496d66934fd5c5a3f7a8247ff4d8d02aeab4682/R/process_response.R#L20-L26

**Comment by wsphd** [comment]
James,

The original contributor to the package has moved on.  I too have issues 
but I can't wait.  It's just easier (in the short-run) to use the a 
Python API.  Over time, I suppose I can use the reticulate package in R 
to coordinate things, but it's still at the end of the day a Python script.

And yes to your other question--retrieving full responses to discussions 
and threads (with author's SIS ID or other) is useful.

Wayne

**Comment by coatless** [comment]
@wsphd Thanks for the fast response. If that's the case, mind if I fork the package? 

I'd like to setup some bindings as UIUC recently started piloting Canvas (https://illinoisedu.instructure.com/)

**Comment by wsphd** [comment]
James,

I think forking is in order.  David's work (he is also at a University) 
was key but I think keeping up with Canvas's large and changing API is, 
frankly, a full-time job in itself.

And Canvas' use is only growing, not shrinking.  Also, some basics from 
Moodle weren't/aren't in Canvas.  We need to make Canvas better (IMHO).

Wayne

**Comment by daranzolin** [comment]
Yes, sorry everyone--rcanvas is no longer relevant to my work, and I'm happy to add maintainers beyond @vanatteveldt. @coatless are you interested? The repo could also hypothetically be transferred.

**Comment by vanatteveldt** [comment]
Hey guys, yeah sorry for not being very active so far. I had plenty of plans, but teaching (and writing a book) got into my way so far. I would be happy to contribute if someone else take's over the lead as well, otherwise I hope to have more time in the coming month.  Whether this is best done as a fork, a transfer, or a github organization I'm not quite sure. 

It might also be good to have a discussion somewhere on what the goals/priorities are that everyone has. I would like to get it on CRAN asap as that would make it a lot easier to get working esp. for mac/windows users, and we can think about other calls that we would like to wrap.
